components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -5,9 +5,9 @@
 
 module ai_chat.mojom;
 
+import "brave/components/ai_chat/core/common/mojom/untrusted_frame.mojom";
```

Comment:
This file can be split up further. I started doing it but stashed the changes for now because I didn't want to further bloat this PR.